core/federated/RTI/rti.c

```diff
@@ -1790,6 +1791,18 @@ int receive_udp_message_and_set_up_clock_sync(int socket_id, uint16_t fed_id) {
     return 1;
 }
 
+/**
+ * Send RTI hello message for authenticated joining of federation.
+ * 
+ * @param socket Socket for the incoming federate tryting to authenticate.
+ */
+void send_rti_hello(int socket) {
```

Comment:
Unclear how this does any authentication.
Seems to just send random bytes.
Comment should explain what a "hello message" is.

Reply:
As same as above, all functions are ready with comments. Thank you for reviewing.